packages/puppeteer-core/src/api/ElementHandle.ts

```diff
@@ -731,59 +732,134 @@ export abstract class ElementHandle<
   }
 
   /**
-   * This method creates and captures a dragevent from the element.
+   * Drags an element to the given element or point.
```

Comment:
Suggestion, as I am unsure about wording as currently my first read went to `drag into` (perform drag&drop).
```suggestion
   * Drags an element to over element or point.
```